app/services/ncr/approval_manager.rb

```diff
@@ -62,7 +62,11 @@ def notify_removed_approvers(original_approvers)
     def ba_6x_approvers
       results = []
 
-      unless work_order.for_whsc_organization?
+      if work_order.for_whsc_organization?
+        # no tier 1
+      elsif work_order.ba_6x_tier1_team?
+        results << Ncr::Mailboxes.ba61_tier1_budgetteam
+      else
```

Comment:
maybe this could be re-written as:

```ruby
if work_order.ba_6x_tier1_team?
   results << Ncr::Mailboxes.ba61_tier1_budgetteam
elsif !work_order.for_whsc_organization?
  results << Ncr::Mailboxes.ba61_tier1_budget
end
```

I am not sure if the logic holds if rewritten this way. But having an `if` condition with a result of `nil` (as it is written currently) does seem unusual to me.